calico_node/startup/startup.go

```diff
@@ -672,6 +672,14 @@ func ensureDefaultConfig(c *client.Client, node *api.Node) error {
 		return err
 	}
 
+	// Configure Felix to allow traffic from the containers to the host (if
```

Comment:
It's not just Mesos deployments is it, it's also k8s no?  More generally though is this basically the default we want for containerized deployments?  If  so probably worth saying that here and that's why we don't do this processing generically in libcalico-go.